Cargo.lock

```diff
@@ -1081,7 +1081,7 @@ dependencies = [
 
 [[package]]
 name = "krill"
-version = "0.15.0-dev"
+version = "0.14.3"
```

Comment:
Shouldn't this be left as 0.15.0-dev?

Reply:
Perhaps it would be more clean to make this change in a separate PR. If you prefer I can do that. But, this is going to be in a bug fix release, so 0.14.3 is more appropriate than 0.15.0.